### PR TITLE
[SQL-DS-CACHE-163][POAE7-1213] update APE doc for Spark in disaggregated mode

### PR DESCRIPTION
**Pull request by yma11**
## What changes were proposed in this pull request?

update APE doc for Spark in disaggregated mode

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by yma11** [comment]
@jikunshang Please help merge this PR.

File: oap-ape/README.md [hunk 1]
```diff
@@ -96,7 +96,27 @@ Extra configuration to enable ape features:
 | `spark.sql.parquet.filterPushdown ` | whether enable filter push down to Parquet native | true |
 | `spark.sql.ape.aggregation.pushdown.enabled` | (Experimental) whether enable aggregation push down to Parquet native | false | 
 |`spark.sql.ape.parquet.cache.enabled` | whether enable parquet cache, you need to start plasma service on worker nodes if you enable this | false |
-|
+|`spark.sql.ape.reader.location remote` | enable remote reader in disaggregated mode | local |
+|`spark.sql.ape.remote.reader.compressed `| enable compression when use remote reader | local |
```
Comment (jikunshang) [inline oap-ape/README.md]: should be true or false?

File: oap-ape/README.md [hunk 2]
```diff
@@ -96,7 +96,27 @@ Extra configuration to enable ape features:
 | `spark.sql.parquet.filterPushdown ` | whether enable filter push down to Parquet native | true |
 | `spark.sql.ape.aggregation.pushdown.enabled` | (Experimental) whether enable aggregation push down to Parquet native | false | 
 |`spark.sql.ape.parquet.cache.enabled` | whether enable parquet cache, you need to start plasma service on worker nodes if you enable this | false |
-|
+|`spark.sql.ape.reader.location remote` | enable remote reader in disaggregated mode | local |
```
Comment (jikunshang) [inline oap-ape/README.md]: option is `local` and `remote` ? Please make it more clear.

**Review by jikunshang — COMMENTED** [review]
Please create an issue and  link it.

File: oap-ape/README.md (hunk 1 above)
Comment (yma11) [reply oap-ape/README.md]: should be false.

File: oap-ape/README.md (hunk 2 above)
Comment (yma11) [reply oap-ape/README.md]: default should be local. Updated.

File: oap-ape/README.md (hunk 1 above)
Comment (yma11) [reply oap-ape/README.md]: Done.

**Pull request merged**